lib/project_types/rails/commands/connect.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Rails
+  module Commands
+    class Connect < ShopifyCli::SubCommand
+      def call(*)
+        if ShopifyCli::Project.has_current? && ShopifyCli::Project.current.env
+          @ctx.puts(@ctx.message('core.connect.production_warning'))
+        end
+
+        org, api_key = ShopifyCli::Commands::Connect.default_connect('rails')
```

Comment:
Is there a particular reason for returning `org` and `api_key` here rather than the app object directly?